jobs.groovy.j2

```diff
@@ -486,10 +486,10 @@ branches.each {
     wrappers {
       timestamps()
       colorizeOutput()
-      {# lock down the multijob to 45 minutes so that 'stuck' jobs won't
+      {# lock down the multijob to 60 minutes so that 'stuck' jobs won't
           block future runs                                                  #}
       timeout {
-          absolute(45)
+          absolute(60)
           }
```

Comment:
the multijob is the main job that executes all the other jobs, it has a larger timeout as it needs to wait for slaves to become available, and to copy artifacts back from the child jobs.